File: C/multiplos.c
#include <stdio.h>

int main() {

    int n1, n2, troca;

    printf("Digite dois numeros inteiros:\n");
    scanf("%d", &n1);
    scanf("%d", &n2);

    if (n1 % n2 == 0 || n2 % n1 == 0) {
        printf("Sao multiplos");
    }
    else {
        printf("Nao sao multiplos");
    }

    return 0;
}

File: Java/Multiplos.java
import java.util.Scanner;

public class Multiplos {
    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);

        int numero1, numero2;

        System.out.println("Digite dois numeros inteiros:");
        numero1 = sc.nextInt();
        numero2 = sc.nextInt();

        if (numero1 % numero2 == 0 || numero2 % numero1 == 0) {
            System.out.println("Sao multiplos");
        }
        else {
            System.out.println("Nao sao multiplos");
        }

        sc.close();
    }
}